packages/target-tools/src/clientInfo.js

```diff
@@ -105,3 +112,78 @@ export const deviceTypeFromUserAgent = userAgent =>
     { name: "Android", regex: /Android [0-9.]+;/ },
     { name: "Desktop", regex: /.*/ }
   ])(userAgent);
+
+/**
+ *
+ * @param {string} browserUAWithVersion Sec-CH-UA or Sec-CH-UA-Full-Version-List
+ * @return {{name: string, version: number}}
+ */
+export const browserFromClientHintsUA = (browserUAWithVersion = "") =>
+  matchUserAgent(
+    [
+      {
+        name: "Chrome",
+        versionGroupIndex: 1,
+        regex: /Chrome";v="([0-9.]+)"/i
+      },
+      {
+        name: "Edge",
+        versionGroupIndex: 1,
+        regex: /Edge";v="([0-9.]+)"/i
+      },
+      {
+        name: "Safari",
```

Comment:
Safari & Firefox don't support Client Hints, how do we test for them ?
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-CH

Reply:
You're right, these regex were added in anticipation of safari and firefox supporting them.  No harm done if they never do.  May need to refine later.